inspirehep/kbs/weblinks.kb

```diff
@@ -0,0 +1,523 @@
+FERMILABEN---Fermilab Library Server (fulltext available)
+FERMILABEXP---Fermilab Library Server (fulltext available)
+FERMILABFN---Fermilab Library Server (fulltext available)
+FERMILABLOI---Fermilab Library Server
+FERMILABLU---Fermilab Library Server (fulltext available)
+FERMILABMASTERS---Fermilab Library Server (fulltext unavailable)
+FERMILABMASTERSF---Fermilab Library Server (fulltext available)
+FERMILABMINERVA---MINERvA Document Server
+FERMILABMISC---Fermilab Library Server (fulltext available)
+FERMILABNAL---Fermilab Library Server (fulltext available)
+FERMILABOTHER---Fermilab Library Server
+FERMILABPROPOSAL---Fermilab Library Server (fulltext available)
+FERMILABPUB---Fermilab Library Server (fulltext available)
+FERMILABRESEARCH---Fermilab Library Server (fulltext available)
+FERMILABSDC---Fermilab Library Server
+FERMILABTEST---Fermilab Library Server (fulltext available)
+FERMILABTHESIS---Fermilab Library Server (fulltext unavailable)
+FERMILABTHESISF---Fermilab Library Server (fulltext available)
+FERMILABTM---Fermilab Library Server (fulltext available)
+FERMILABTODAY---Fermilab Today Result of the Week
+FERMILABUPC---Fermilab Library Server (fulltext available)
+FERMILABVLHCPUB---Fermilab Library Server (fulltext available)
+FERMILABWORKBOOK---Fermilab Library Server
+FIZIKAB---Fizika B Server
+FNDPA---Found.Phys. Server
+FOCUS---Physical Review Focus
+FPLEE---Found. Phys. Lett. Server
+FPYKA---Fortschr. Phys. Server
+FZKAA-B---Fizika B Server
+FZKAAB---Fizika B Server
+GRGVA---Gen.Rel.Grav. Server
+HEPLW---HEP Lib.Webzine Server
+HEPPDF---PDF Server
+HLTPA---Health Physics Server
+HSERVER---HTML_Version from a server
+HTTP://POS.SISSA.IT/ARCHIVE/CONFERENCES/045/026/LHC07_026.PDF---HTTP://WWW-BD.FNAL.GOV/ICFABD/NEWSLETTER45.PDF
+ICTP---ICTP Trieste Preprint Server
+ICTP-LNS---ICTP Lecture Notes Server
+IEEE---IEEExplore Server
+IHEP---IHEP Document Server
+IJMPA---Int. J. Mod. Phys. Server
+IJMPB---Int. J. Mod. Phys. Server
+IJMPC---Int. J. Mod. Phys. Server
+IJMPD---Int. J. Mod. Phys. Server
+IJMPE---Int. J. Mod. Phys. Server
+IJTPB---Int. J. Theor. Phys. Server
+IMPAE---Int.J.Mod.Phys.A Server
+IMPAE-A---Int.J.Mod.Phys.A Server
+IMPAE-B---Int.J.Mod.Phys.B Server
+IMPAE-C---Int.J.Mod.Phys.C Server
+IMPAE-D---Int.J.Mod.Phys.D Server
+IMPAE-E---Int.J.Mod.Phys.E Server
+IN2P3---HAL in2p3 Server
+INDICO---CERN Indico Server
+INETA---Instruments and Experimental Techniques Server
+INTERACTIONS---Interactions.org article
+IOPLETT---IOP Phys.Express Lett. Server
+IRNPE---Int.Rev.Nucl.Phys. Server
+JACOW---Full-text at JACoW Server
+JACOWS---Slides on JACoW Server
+JAUMA---J.Austral.Math.Soc. Server
+JCAPA---JCAP Electronic Journal Server
+JCTPA---J. Comput. Phys. Server
+JDGEA---J. Diff. Geom. Server
+JETP---J. Exp. Theor. Phys. Server (Access may be <a href=/find/restrict.html> restricted )
+JETPL---J. Exp. Theor. Phys. Lett. Server (Access may be <a href=/find/restrict.html> restricted )
```

Comment:
@annetteholtkamp also mention this KB is a bit old style. We can really drop all the `(Access may be...)` since anyway our users don't expect INSPIRE will magically grant them access to papers. Also more and more we should use DOIs to redirect user to papers.